Adyen/Model/Checkout/Action/CheckoutBankTransferAction.cs

```diff
@@ -29,13 +29,13 @@
 using System.Text;
 using Newtonsoft.Json;
 
-namespace Adyen.Model.Checkout
+namespace Adyen.Model.Checkout.Action
 {
     /// <summary>
     /// CheckoutBankTransferAction
     /// </summary>
     [DataContract]
-    public partial class CheckoutBankTransferAction : IEquatable<CheckoutBankTransferAction>, IValidatableObject
+    public partial class CheckoutBankTransferAction : IEquatable<CheckoutBankTransferAction>, IValidatableObject, IPaymentResponseAction
     {
         /// <summary>
         /// Initializes a new instance of the <see cref="CheckoutBankTransferAction" /> class.
```

Comment:
https://github.com/Adyen/adyen-openapi/blob/master/json/CheckoutService-v67.json#L3582

I think we can remove `paymentData` from this class

Reply:
Indeed [commit](https://github.com/Adyen/adyen-dotnet-api-library/pull/434/commits/2a97c2cebb1135b099ca6afd3ca665381fdd2651)